src/lib/orionTypes/EntityTypeVector.cpp

```diff
@@ -110,11 +110,11 @@ std::string TypeEntityVector::check
 
 /* ****************************************************************************
 *
-* TypeEntityVector::present -
+* EntityTypeVector::present -
 */
-void TypeEntityVector::present(const std::string& indent)
+void EntityTypeVector::present(const std::string& indent)
 {
-  LM_F(("%lu TypeEntitys", (uint64_t) vec.size()));
+  LM_F(("%lu EntityTypeVector", (uint64_t) vec.size()));
```

Comment:
Here the text should be **EntityTypes** - it expresses the number of items in the vector (the previous text was incorrect also ...  *TypeEntities*)